src/main/scala/com/test/MainZIO.scala

```diff
@@ -1,23 +1,22 @@
 package com.test
 
-import cats.effect.Blocker
+import cats.effect.Async
+import cats.effect.std.Dispatcher
 import zio._
 import zio.interop.catz._
 import zio.interop.catz.implicits._
 import zio.blocking.Blocking
 import zio.clock.Clock
 import org.http4s.blaze.server.BlazeServerBuilder
 
-import scala.concurrent.ExecutionContext.global
-
 object MainZIO extends App {
 
-  def program(implicit runtime: zio.Runtime[Clock & Blocking]): Task[ExitCode] =
-      Blocker[Task].use { b =>
-        BlazeServerBuilder[Task](global)
+  def program(implicit runtime: zio.Runtime[Clock & Blocking], async: Async[Task]): Task[ExitCode] =
+      Dispatcher[Task].use { d =>
```

Comment:
Would keep it w/o Dispatcher and Async requirements. For me it looks cleaner, when is used with ZIO - staying as close to ZIO w/o additions as long as possible and switching to CE3 typeclasses only where necessary.

Reply:
i never used ZIO before, i'll take a look into it to fix that, sorry ^^"